File: projet XAMP/classes/Membre.php
<?php
class Membre {
    private $id, $nom, $prenom, $ville, $genre;

    function __construct($id, $nom, $prenom, $ville, $genre) {
        $this->id     = $id;
        $this->nom    = $nom;
        $this->prenom = $prenom;
        $this->ville  = $ville;
        $this->genre  = $genre;
    }

    function getNom()    { return $this->nom; }
    function getPrenom() { return $this->prenom; }
    function getVille()  { return $this->ville; }
    function getGenre()  { return $this->genre; }
}
?>

File: app/src/main/java/com/example/lab_9_sas_houda/beans/Membre.java
package com.example.lab_9_sas_houda.beans;

public class Membre {
    private int id;
    private String nom, prenom, ville, genre;

    public Membre() {}

    public Membre(int id, String nom, String prenom,
                  String ville, String genre) {
        this.id     = id;
        this.nom    = nom;
        this.prenom = prenom;
        this.ville  = ville;
        this.genre  = genre;
    }

    public int getId()       { return id; }
    public String getNom()   { return nom; }
    public String getPrenom(){ return prenom; }
    public String getVille() { return ville; }
    public String getGenre() { return genre; }

    @Override
    public String toString() {
        return "Membre{id=" + id +
                ", nom='" + nom + '\'' +
                ", prenom='" + prenom + '\'' +
                ", ville='" + ville + '\'' +
                ", genre='" + genre + '\'' + '}';
    }
}